packages/eslint-plugin/src/rules/lines-between-class-members.ts

```diff
@@ -32,7 +32,10 @@ export default util.createRule<Options, MessageIds>({
     },
     fixable: 'whitespace',
     schema,
-    messages: baseRule.meta.messages,
+    messages: baseRule.meta.messages ?? {
+      never: 'Unexpected blank line between class members.',
+      always: 'Expected blank line between class members.',
+    },
```

Comment:
this rule has been using message ids since 5.12.
I guess we should keep this alive until we drop v5.0.0 from the supported ranges.